src/messages.cs

```diff
@@ -46,11 +48,23 @@ public enum MSMessageErrorCode : nint
 		StickerFileImproperFileSize,
 		StickerFileImproperFileFormat,
 		UrlExceedsMaxSize,
+		SendWithoutRecentInteraction,
+		SendWhileNotVisible,
+	}
+
+	[iOS (11,0)]
+	[Protocol]
+	interface MSMessagesAppTranscriptPresentation
+	{
+		[iOS (11,0)]
+		[Abstract]
+		[Export ("contentSizeThatFits:")]
+		CGSize ContentSizeThatFits (CGSize size);
 	}
 
 	[iOS (10,0)]
 	[BaseType (typeof(UIViewController))]
-	interface MSMessagesAppViewController
+	interface MSMessagesAppViewController : MSMessagesAppTranscriptPresentation
```

Comment:
Deriving from the protocol will inline all the protocol members, which IMO is not what you want to do here.

Derive from `IMSMessagesAppTranscriptPresentation` instead (you'll have to add an empty `interface IMSMessagesAppTranscriptPresentation {}` declaration first). That will tell the generator that the class implements the protocol (and generate in managed code implement the corresponding protocol interface), without inlining the protocol members.